xmodule/course_module.py

```diff
@@ -403,14 +403,6 @@ class CourseFields:  # lint-amnesty, pylint: disable=missing-class-docstring
         ),
         scope=Scope.settings
     )
-    discussion_sort_alpha = Boolean(
```

Comment:
this will remove the field from advanced settings?

Reply:
Yes